vmware/ngc/NGC-Register/src/main/resources/static/config/PluginInfo.properties

```diff
@@ -0,0 +1,10 @@
+storage.plugin.registvis.info=\u4fe1\u606f
+plugin.name=OpenSDS NGC Plugin
+plugin.key=org.opensds.vmware.ngc
+plugin.summary=OpenSDS Next Generation Center Plugin
```

Comment:
Next Generation Center Plugin OR Next Generation VMWare vCenter Plugin ?

Reply:
Changed to Next Generation VMWare vCenter Plugin.